src/Domain/Repositories/GetMembershipApplicationException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Domain\Repositories;
+
+/**
+ * @license GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class GetMembershipApplicationException extends \RuntimeException {
```

Comment:
This code made me wonder if the Liskov Substitution Principle applies to constructors, and found the "right" answer on StackOverflow: http://stackoverflow.com/questions/5490824/should-constructors-comply-with-the-liskov-substitution-principle

Might be worth pointing out some times.